sql/query/expr.go

```diff
@@ -312,6 +312,8 @@ func evalComparisonExpr(expr *parser.ComparisonExpr, env Env) (sqlwire.Datum, er
 			v = *left.FloatVal == *right.FloatVal
 		case stringOp:
 			v = *left.StringVal == *right.StringVal
+		default:
+			panic(fmt.Errorf("unsupported op type: %d", typ))
```

Comment:
Instead of `fmt.Errorf`, use either `fmt.Sprintf` (`panic` doesn't care whether it gets an `error` or `string`) or `util.Errorf`.